projects/Mallard/src/components/navigator/background.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react'
+
+import Svg, { Circle, Line } from 'react-native-svg'
+
+const Background = ({
+    fill,
+    stops,
+    height,
+    radius,
+}: {
+    fill: string
+    stops: number
+    height: number
+    radius: number
+}) => {
+    const Stop = ({ fill, ...props }: { fill: string; [key: string]: any }) => {
+        return <Circle r={radius} cy={height} fill={fill} {...props} />
+    }
+    const stopElements = []
```

Comment:
`Array.new(stops - 1).map` if you preferred?

Reply:
You mean `new Array(stops-1)` right? Or did I miss a new Array constructor?? 

you can't map over an empty array even if it has a length (odd) so you need to run `new Array(4).fill().map()` which will fill it with undefineds first and it feels a bit wasteful :x like the proper way to do this should be a function that fills it properly in the first place but there's not one